.storybook/webpack.config.js

```diff
@@ -0,0 +1,24 @@
+module.exports = ({ config }) => {
+  config.resolve.modules = config.resolve.modules.filter((m) => m !== '.');
+  config.module.rules = config.module.rules.filter((rule) => {
+    return !(
+      rule.use &&
+      rule.use[0].loader &&
+      rule.use[0].loader.includes('eslint-loader')
+    );
+  });
+  config.module.rules.push({
+    test: /\.(ts|tsx)$/,
+    use: [
+      {
+        loader: require.resolve('babel-loader'),
+        options: {
+          presets: [['react-app', { flow: false, typescript: true }]],
+        },
+      },
+      require.resolve('react-docgen-typescript-loader'),
+    ],
+  });
+  config.resolve.extensions.push('.ts', '.tsx');
+  return config;
+};
```

Comment:
What are we doing here that was missing before?

Reply:
The `react-docgen-typescript-loader` was missing before, and is needed in order to make the `@storybook/addon-info` plugin work.

Reply:
Ok because the addon-info pulls info from TS source right?

By defining this config is a default not applied anymore? Because there's a lot there besides the one loader addition.

Just trying to understand.

Reply:
This webpack config is merged with the default storybook webpack config. So anything in this file should be seen as an incremental override. Frankly, the rabbit hole of research I did on how to get the react-docgen-typescript-loader to work _alongside_ our existing create-react-app storybook configuration led me to this implementation, but I didn't do enough research to determine what we actually needed. 

Check it out now, I think I just simplified it significantly, it looks like there was a lot of unnecessary stuff in there, just based on trial-and-error deleting and reloading storybook after removing line-by-line.

Reply:
Much better 🤩 Thanks!